wiki/Resnap/en.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Resnap
+
+**Resnap** refers to a pair of commands under the `Timing` header in the [beatmap editor](/wiki/Beatmap_Editor). One of which applies to the entire [beatmap](/wiki/Beatmaps) and the other to the current timing section (if the beatmap has only one timing section, both commands are the same).
+
+This command will automatically reposition all [hit objects](/wiki/Hit_object) on the timeline so that they fall on the closest beat division your snap divisor allows. Usage is suggested when there has been a change in offset and/or BPM, after objects have been placed. It is important to check every hit object and adjust its position on the [timeline](/wiki/Beatmap_Editor/Timelines) manually, since this tool isn't fully accurate.
```

Comment:
```suggestion
This command will automatically reposition all [hit objects](/wiki/Hit_object) on the timeline so that they fall on the closest beat division that the snap divisor allows. Usage is suggested when there has been a change in offset and/or BPM, after objects have been placed. Many creators reccomend checking every hit object and adjusting its position on the [timeline](/wiki/Beatmap_Editor/Timelines) manually due to this tool's inaccurate and systematic nature.
```